markdown/JekyllMarkdown/main.swift

```diff
@@ -0,0 +1,114 @@
+import Foundation
+import Markup
+
+func main() {
+    guard let (from, to, permalink) = arguments() else { Console.help.show(); exit(-1) }
+    renderJekyll(from: from, to: to, permalink: permalink)
+}
+
+
+/// Method to render a page into Jekyll format.
+///
+/// - Parameters:
+///   - filePath: input page in Apple's playgorund format.
+///   - outputPath: output where to write the Jekyll render.
+///   - permalink: website's relative url where locate the page.
+func renderJekyll(from filePath: String, to outputPath: String, permalink: String) {
+    let fileURL = URL(fileURLWithPath: filePath)
+    let outputURL = URL(fileURLWithPath: outputPath)
+
+    guard let content = try? String(contentsOf: fileURL, encoding: .utf8),
+          let rendered = JekyllGenerator(permalink: permalink).render(content: content),
+          let _ = try? rendered.write(to: outputURL, atomically: true, encoding: .utf8) else { Console.error.show(); return }
+
+    Console.success.show()
+}
+
+
+/// Console output
+///
+/// - error: show general error. The script fails.
+/// - success: show general success. The script finishes successfully.
+/// - help: show the help. How to use this script.
+enum Console {
+    case error
+    case success
+    case help
+
+    func show() {
+        switch self {
+        case .error: printError()
+        case .success: printSuccess()
+        case .help: printHelp()
+        }
+    }
+
+    private func printError() {
+        print("ERROR")
```

Comment:
Shouldn't we provide more info about this? Same in `printSuccess` and `printHelp`.

Reply:
Yep, sorry, I have not finished this.